packages/amplication-client/src/Resource/ResourceHome.tsx

```diff
@@ -18,17 +16,22 @@ import OverviewTile from "./OverviewTile";
 import RolesTile from "./RolesTile";
 import SyncWithGithubTile from "./SyncWithGithubTile";
 import ViewCodeViewTile from "./ViewCodeViewTile";
+import { AppRouteProps } from "../routes/routesUtil";
 
-type Props = {
-  match: match<{ resource: string }>;
+
+
+type Props = AppRouteProps & {
+  match: match<{
+    workspace: string;
+    project: string;
+    resource: string; 
+  }>;
 };
 
 const CLASS_NAME = "resource-home";
```

Comment:
should we use moduleClass?

Reply:
@yuval-hazaz  do we now make all the refactors needed or only focus on fix the route issues??

Reply:
you are correct